opm/input/eclipse/share/keywords/000_Eclipse100/B/BLOCK_PROBE

```diff
@@ -54,6 +54,9 @@
     "BVELGJ",
     "BVELGK",
     "BPR",
+    "BGCDI",
+    "BGCDM",
+    "BWCD",
```

Comment:
These are OPM-specific vectors and should therefore be listed in [900_OPM/B/BLOCK_PROBE_900](https://github.com/OPM/opm-common/blob/0323f098413a480b1f29aa7eb33bae77ffa2be20/opm/input/eclipse/share/keywords/900_OPM/B/BLOCK_PROBE900).

Reply:
Thanks, now they are moved to the OPM-specific files.